### ISSUE TITLE
This order can't be paid with paypal

### ISSUE THREAD

**Issue by NicolaConnect** [ovh/php-ovh#65]
Hello,
My problem may not be strictly related to this package, but maybe someone can help. I am currently working on an interface for my team to create new domains, request new service, etc...
Everything is ok but payment, my current workflow is:
1. Create a cart
2. Add new services
3. Set required config fields
4. Assign the cart to logged in user
5. Checkout
6. Try to pay via PayPal

At this point, I get an error. The error is:
`403 ActionImpossible This order can't be paid with paypal`

My request is as follow:
`$request = [
	"paymentMean" => "paypal",
	"paymentMeanId" => XXXXXXX
];`
sent to
`/me/order/MY_ORDER_ID/payWithRegisteredPaymentMean`

I have a valid PayPal account attached to my OVH account and it is used for auto-renewals through OVH. Can I use it via API?

Thanks for your time!

**Comment by Rbeuque74** [comment]
Hello @NicolaConnect,
Can you contact api@ml.ovh.net for this kind of errors ? We will be glad to help you resolve your issue on this mailing-list.
Also, if you can give us an orderId on the mailing-list so the processing of your problem will be faster ;)

Thanks,
Romain

**Comment by Hesesses** [comment]
I tried to post api@ml.ovh.net and got this back:
```
Hi. This is the qmail-send program at mx1.ovh.net.
I'm afraid I wasn't able to deliver your message to the following addresses.
This is a permanent error; I've given up. Sorry it didn't work out.

<ml.ovh.net-api@ml.ovh.net>:
Sorry, only subscribers may post. If you are a subscriber, please forward this message to api-owner@ml.ovh.net to get your new address included (#5.7.2)
```

**Comment by Rbeuque74** [comment]
You have to subscribe using:

api-subscribe < a t > ml.ovh.net
